proto/shared_constants.proto

```diff
@@ -190,6 +190,12 @@ message PortBehavior {
     down = 0;
     up = 1;
   }
+
+  enum AutoSequestering {
+    default = 0;
+    true = 1;
```

Comment:
I think it'll be too confusing to use "true"/"false" as enum constants... so I'd posit these should be representative of the function... so something like "ENABLED" or "SKIP" respectively -- whatever.  But, I think in this context boolean == bad.

Reply:
I can change these to enabled and disabled but I think the default_auto_sequester should still be a bool. See reasoning below...